<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>shot-create</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="selectWindow">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>40</y>
      <width>911</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="shot">
     <attribute name="title">
      <string>Add Shot</string>
     </attribute>
     <widget class="QListWidget" name="scenes">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>256</width>
        <height>271</height>
       </rect>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="autoScroll">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
      <property name="resizeMode">
       <enum>QListView::Fixed</enum>
      </property>
      <property name="uniformItemSizes">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="currentRow">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Sequences</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_seq_btn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>360</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add New Sequence</string>
      </property>
     </widget>
     <widget class="QListWidget" name="shots">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>40</y>
        <width>271</width>
        <height>361</height>
       </rect>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="autoScrollMargin">
       <number>21</number>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>20</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Shots</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_shot_btn">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>170</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="shot_title">
      <property name="geometry">
       <rect>
        <x>419</x>
        <y>142</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>146</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>New Shot:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>62</x>
        <y>337</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>New Sequence:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="new_sqn_title">
      <property name="geometry">
       <rect>
        <x>159</x>
        <y>336</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="assets">
     <attribute name="title">
      <string>Add Assets</string>
     </attribute>
     <widget class="QListWidget" name="shot_assets_view">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>40</y>
        <width>256</width>
        <height>431</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
     <widget class="QPushButton" name="add_asset_btn">
      <property name="geometry">
       <rect>
        <x>408</x>
        <y>320</y>
        <width>110</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add  &gt;&gt;</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="load">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>281</width>
        <height>441</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="characters">
       <attribute name="title">
        <string>Characters</string>
       </attribute>
       <widget class="QTreeView" name="char_asset_dir_view">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>271</width>
          <height>411</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="indentation">
         <number>10</number>
        </property>
        <attribute name="headerCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="headerMinimumSectionSize">
         <number>4</number>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
       </widget>
      </widget>
      <widget class="QWidget" name="environments">
       <attribute name="title">
        <string>Environments</string>
       </attribute>
       <widget class="QTreeView" name="env_asset_dir_view">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>271</width>
          <height>411</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="indentation">
         <number>10</number>
        </property>
        <attribute name="headerCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="headerMinimumSectionSize">
         <number>4</number>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
       </widget>
      </widget>
      <widget class="QWidget" name="props">
       <attribute name="title">
        <string>Props</string>
       </attribute>
       <widget class="QTreeView" name="prop_asset_dir_view">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>271</width>
          <height>411</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="indentation">
         <number>10</number>
        </property>
        <attribute name="headerCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="remove_asset_btn">
      <property name="geometry">
       <rect>
        <x>408</x>
        <y>360</y>
        <width>110</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;&lt;  Remove</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>50</y>
        <width>56</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Preview</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Available Assets</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>10</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Imported Assets</string>
      </property>
     </widget>
     <widget class="QToolButton" name="preview_right_btn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>50</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="arrowType">
       <enum>Qt::RightArrow</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="preview_left_btn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>50</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
      <property name="arrowType">
       <enum>Qt::LeftArrow</enum>
      </property>
     </widget>
     <widget class="QLabel" name="preview">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>70</y>
        <width>271</width>
        <height>211</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : white;
border: 1px solid grey</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>shot_assets_view</zorder>
     <zorder>add_asset_btn</zorder>
     <zorder>load</zorder>
     <zorder>remove_asset_btn</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>preview_left_btn</zorder>
     <zorder>preview_right_btn</zorder>
     <zorder>preview</zorder>
    </widget>
    <widget class="QWidget" name="camera">
     <attribute name="title">
      <string>Add Settings</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>86</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Lens Focal Length:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="focal_length">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>82</y>
        <width>104</width>
        <height>26</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>24</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string notr="true">35</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>50</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>85</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>135</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>200</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>52</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Camera Settings</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>398</x>
        <y>116</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>F-Stop:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="fstop">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>112</y>
        <width>104</width>
        <height>26</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <item>
       <property name="text">
        <string>1.4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2.8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5.6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>32</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>150</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Shutter Angle:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="shutter_angle">
      <property name="geometry">
       <rect>
        <x>454</x>
        <y>146</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="specialValueText">
       <string/>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>359</number>
      </property>
      <property name="value">
       <number>180</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Project: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sequence:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Shot:</string>
    </property>
   </widget>
   <widget class="QLabel" name="project_lb">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sqn_lb">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="shot_lb">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="build_btn">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>550</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Build</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_btn">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>550</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>921</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_project"/>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionQuit_shot_create"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_project">
   <property name="text">
    <string>Open project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionQuit_shot_create">
   <property name="text">
    <string>Quit shot-create</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
